adapters/pancakeswap/src/index.ts

```diff
@@ -78,7 +80,27 @@ export const getUserTVLByBlock = async ({
         position.token1.balance;
   }
 
+  // Get sickles and their owners
+  const sickleAddresses = await getSickles();
+  const sickleOwners = await getSickleOwners(sickleAddresses.map(s => s.sickle));
+
+  // Replace sickle addresses with their owners
+  const updatedBalances: Record<string, Record<string, bigint>> = {};
   for (const [user, tokenBalances] of Object.entries(balances)) {
+    const owner = (sickleOwners as Record<string, string>)[user] || user; // Replace sickle address with owner address
```

Comment:
sickleOwners key address is checksummed right? better convert it to lowercase, as balances[user] is lowercase by default in subgraph